schema.sql

```diff
@@ -69,7 +69,11 @@ CREATE TABLE genotypes (
        quality TEXT,
 
        -- Cycledash-derived data
-       "annotations:gene_names" TEXT,
+       -- These are from the varcode annotation worker
+       "annotations:gene_name" TEXT,
```

Comment:
If there are multiple genes, do they still get displayed? (If so, this should probably still be "annotations:gene_name**s**")